detection_rules/etc/stack-schema-map.yaml

```diff
@@ -41,5 +41,9 @@
   ecs: "8.2.1"
 
 "8.4.0":
-  beats: "main"  # TODO: update this once beats releases
+  beats: "main"
+  ecs: "8.2.1"
+
+"8.5.0":
+  beats: "main"
```

Comment:
Good catch, I think this may be 8.3.3 then if we are looking at tag releases from github.com/elastic/beats.

@brokensound77 -> Where should this information be pulled from, the repo itself? I'll update the documentation if so.

Reply:
similar to #1995 - so just point to the latest 8.3.x for beats and ecs. Then you can update 8.5 and main to use them until 8.4 is cut